5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
The structure within the `image-files` directory is the same as they'll land on the image.

So `/usr/local/bin/docker-composer-install`, according to https://github.com/docker-library/php/blob/master/7.1/Dockerfile#L129 - or `opt` or `install-scripts`